### ISSUE TITLE
Convert nbpresent to use nbsetuptools

### ISSUE THREAD

**Issue by ijstokes** [Anaconda-Server/anaconda-nb-extensions#105]
`nbpresent` should be built on `nbsetuptools` but to do so will mean having PyPI packages for `nbsetuptools` and its dependencies.  This requirement is being covered in #104.  Once that is complete then @mmarchetti says he already has a PR on `nbpresent` that will use `nbsetuptools` so the foundational work for this is done, it is just that the dependencies need to be available in PyPI and then the updated `nbpresent` PR merged.

**Comment by ijstokes** [comment]
@mmarchetti if what I'm saying is true, can you (or have you already?) created a PR on `nbpresent` with your version that uses `nbsetuptools` and `nbwrapper`?

**Comment by mmarchetti** [comment]
I pushed a branch named `use_nbsetuptools` containing almost all of the required changes. The last thing needed is a `main.js` that loads `nbpresent.min.js` (nbsetuptools assumes the notebook part of the extension lives in main.js). I've done that locally and tested it, however, the `static` directory lives in a submodule tied to another commit and I'm not sure how to update it.

**Comment by damianavila** [comment]
> the static directory lives in a submodule tied to another commit and I'm not sure how to update it.

yep... that's an issue... @bollwyvl thoughts? Is it a requirement for you to use submodules?

**Comment by bollwyvl** [comment]
That's a developer convenience so I can deploy to gh-pages for
bookmarklet/fallback.

I could drop the submodule, and just have a local git repo in that place.
It is entirely regenerated every time you run `npm run dist`

**Comment by damianavila** [comment]
>>It is entirely regenerated every time you run `npm run dist`

@mmarchetti ^^^

No need to drop it for now... I think...

**Comment by stephenakearns** [comment]
Hi @bollwyvl - just reviewing our remaining tasks on the burndown - checking to see the status of this one - did it get completed?

**Comment by stephenakearns** [comment]
Task still outstanding, but on hold. nbPresent available to Pip install, but to move it into a conda package, we need to release all the dependencies on PyPI. Revisit this in the future.

**Comment by damianavila** [comment]
This was done some time in the past. Closing.
